### PR TITLE
update: change devops api from devops to devopsprojects

### PR DESCRIPTION
**Pull request by harrisonliu5**
**What type of PR is this?**

/kind api-change

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for reviewers**:
```
```

**Additional documentation, usage docs, etc.**:
```docs

```

### CODE REVIEW

File: src/pages/console/containers/Dashboard/User/Resources/index.jsx [hunk 1]
```diff
@@ -173,8 +173,9 @@ export default class Resources extends React.Component {
         .create(data, { workspace: this.workspace })
         .then(result => {
           this.hideCreateModal()
-          if (result.project_id) {
-            this.routing.push(`/devops/${result.project_id}`)
+          if (result.name) {
+            // this.routing.push(`/devops/${result.project_id}`)
```
Comment (leoendless) [inline src/pages/console/containers/Dashboard/User/Resources/index.jsx:177]: Please remove the code comment, you can find the history code in git.

Comment (harrisonliu5) [reply src/pages/console/containers/Dashboard/User/Resources/index.jsx]: The lastest commit is removed

**Pull request merged**